src/applications/disability-benefits/all-claims/tests/pages/evidenceTypes.unit.spec.jsx

```diff
@@ -22,75 +27,82 @@ describe('evidenceTypes', () => {
       <DefinitionTester
         definitions={formConfig.defaultDefinitions}
         schema={schema}
         uiSchema={uiSchema}
-        data={{
-          'view:hasEvidence': false,
-        }}
+        data={{}}
         formData={{}}
         onSubmit={onSubmit}
       />,
     );
 
-    form.find('form').simulate('submit');
+    $('va-radio', container).__events.vaValueChange({
+      detail: { value: 'N' },
+    });
+    const submitButton = getByText(/submit/i);
+    userEvent.click(submitButton);
+    expect($('va-radio').error).to.be.null;
     expect(onSubmit.calledOnce).to.be.true;
-    expect(form.find('.usa-input-error-message').length).to.equal(0);
-    form.unmount();
   });
 
   it('should require at least one evidence type when evidence selected', () => {
     const onSubmit = sinon.spy();
-    const form = mount(
+    const { getByText, container } = render(
       <DefinitionTester
         definitions={formConfig.defaultDefinitions}
         schema={schema}
         uiSchema={uiSchema}
-        data={{
-          'view:hasEvidence': true,
-        }}
+        data={{}}
         formData={{}}
         onSubmit={onSubmit}
       />,
     );
 
-    form.find('form').simulate('submit');
+    $('va-radio', container).__events.vaValueChange({
+      detail: { value: 'Y' },
+    });
+    const submitButton = getByText(/submit/i);
+    userEvent.click(submitButton);
+    const error = getByText(
+      /please select at least one type of supporting evidence/i,
+    );
+    expect(error).to.exist;
     expect(onSubmit.called).to.be.false;
-    expect(form.find('.usa-input-error-message').length).to.equal(1);
-    form.unmount();
   });
 
   it('should submit with all required info', () => {
     const onSubmit = sinon.spy();
-    const form = mount(
+    const { getByText, getByLabelText, container } = render(
       <DefinitionTester
         definitions={formConfig.defaultDefinitions}
         schema={schema}
         uiSchema={uiSchema}
-        data={{
-          'view:hasEvidence': true,
-          'view:hasEvidenceFollowUp': {
-            'view:selectableEvidenceTypes': {
-              'view:hasVaMedicalRecords': true,
-            },
-          },
-        }}
+        data={{}}
         formData={{}}
         onSubmit={onSubmit}
       />,
     );
 
-    form.find('form').simulate('submit');
+    $('va-radio', container).__events.vaValueChange({
+      detail: { value: 'Y' },
+    });
+    const checkbox = getByLabelText(/va medical records/i);
+    userEvent.click(checkbox);
+    const submitButton = getByText(/submit/i);
+    userEvent.click(submitButton);
     expect(onSubmit.calledOnce).to.be.true;
-    expect(form.find('.usa-input-error-message').length).to.equal(0);
-    form.unmount();
+    try {
+      getByText(/please select at least one type of supporting evidence/i);
+    } catch (e) {
+      // Error text in try block should not be present.
+      expect(e).to.exist;
+    }
```

Comment:
in the previous test this was used `expect($('va-radio').error).to.be.null`. could we use that again?